src/electron/electron.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @author James Oswald
+ * The Electron Main File, runs as the master Node.js process
+ * Manages the application window.
+ */
+
+// eslint-disable-next-line
+const {app, BrowserWindow} = require("electron");
+const path = require("path");
+const url = require("url");
+
+function createWindow() {
+    // Create the browser window.
+    const win = new BrowserWindow({
+        width: 800,
```

Comment:
Currently this size causes components to be squished. Please increase the width to 1200 or more.

Reply:
Should be fixed on the renderer (CSS) side. Will be handled by #42.